app/validators/valid_govuk_path_validator.rb

```diff
@@ -0,0 +1,8 @@
+class ValidGovukPathValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    raise URI::InvalidURIError unless value.starts_with?('/')
+    Services.content_store.content_item!(value)
+  rescue GdsApi::ContentStore::ItemNotFound, URI::InvalidURIError
+    record.errors[attribute] << "This URL doesn't exist on GOV.UK."
```

Comment:
This message isn't quite right - things like attachments exist on GOV.UK, though we don't want to allow redirecting to them.  Perhaps "This URL isn't a valid target for a redirect on GOV.UK"?